### ISSUE TITLE
Ziwft Virtual Gearing - Tacx Neo2 - Flywheel Spins in gear 6 >1

### ISSUE THREAD

**Issue by me91st** [cagnulein/qdomyos-zwift#2907]
[debug-Fri_Dec_13_19_08_06_2024.log](https://github.com/user-attachments/files/18128089/debug-Fri_Dec_13_19_08_06_2024.log)
Using Zwift virtual gears, windows PC with the app running on IOS.  In gear selection 6 >1 a small pedal causes the flywheel on the NEO2 to ramp up to a very high speed, this casues no resistance on Zwift in these gears. You can only slows and stop the NEO if you then shift up to a lower gear (around gear 10)

Desktop
 - OS: Windows 11
 - Version 26100.2605

Smartphone
 - Device: iPhone 14
 - OS: 18.2
 - Version 2.18

**Comment by cagnulein** [comment]
did you try the gears gain to 0.5 or even less? that's the cause of this
because the value 1 is  probably too strong for you trainer
let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 13 dic 2024 alle ore 16:22 me91st ***@***.***>
ha scritto:

**Comment by me91st** [comment]
Under advanced I can see:

Watt Gain
Speed Gain
Cadence Gain
Inclination Gain

But no gears gain?  Is it the Watt gain I should try?

**Comment by cagnulein** [comment]
Sorry under bike options

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno sab 14 dic 2024 alle 06:08 me91st ***@***.***> ha
scritto:

**Comment by me91st** [comment]
Hi

Not yet, I will test later today.

**Comment by me91st** [comment]
Hi

I tested; it reduces the rate at which the NEO flywheel starts to go into crazy spin mode, but at 0.5 you loose top end gears.

With the setting of Gears Gain at 1, the rider does move as expected, and the virtual gears are good, but the disconcerting thing is the NEO is spinning the flywheel at a constant high speed once you have used gears (6>1), even if you are not pedalling.  Only clicking up to a gear of around 12 will start to slow the flywheel down to a stop.

**Comment by cagnulein** [comment]
I didn't quite get it. Are you telling that you can use only higher gear so? High gear are small enough to have a smooth gearing?

**Comment by eskimos20** [comment]
I can answer since I have the same problem. When Zwift starts default gear for me is 8. If I Use gear >=8 all works as supposed to. 

But if I am pedling at any watt/speed and choose gears <8 and stop pedling then the flywheel on the Tacx Neo2 will continue to spin forever. If I drop gears from 7 to 6 to 5 and so on.. the flywheel will spin faster. If I go back to 8 it will stop after a couple of seconds.

**Comment by cagnulein** [comment]
Yes because lower than 8 means using a negative gear gain to the trainer so like found downhill at resistance level

**Comment by eskimos20** [comment]
Ok. Maybe then an 10-15% uphill will be useful for gears below 8 if running z1-z2 power :) Thanks for the anwer.

**Comment by cagnulein** [comment]
For sure :)

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 30 dic 2024 alle 19:08 eskimos20 ***@***.***>
ha scritto:
